cmd/airflow.go

```diff
@@ -144,16 +151,13 @@ func newDevRootCmd(platformCoreClient astroplatformcore.CoreClient, astroCoreCli
 
 func newAirflowInitCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "init",
-		Short:   "Create a new Astro project in your working directory",
-		Long:    "Create a new Astro project in your working directory. This generates the files you need to start an Airflow environment on your local machine and deploy your project to a Deployment on Astro or Astronomer Software.",
-		Example: initCloudExample,
-		Args:    cobra.MaximumNArgs(1),
-		// ignore PersistentPreRunE of root command
-		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-			return nil
-		},
```

Comment:
All of these blocks are have been replaced with the same function - `DoNothing`.